app/models/miq_server/server_smart_proxy.rb

```diff
@@ -69,26 +69,12 @@ def queue_call(ost)
     case ost.method_name
     when "scan_metadata", "sync_metadata"
       worker_setting = MiqSmartProxyWorker.worker_settings
-      #
-      # TODO: until we get location/offset read capability for OpenStack
-      # image data, OpenStack fleecing is prone to timeout (based on image size).
-      # We try to adjust the timeout here - maybe this should be calculated based
-      # on the size of the image, but that information isn't directly available.
-      #
-      timeout_adj = 1
+
+      timeout_adjustment_multiplier = VmOrTemplate.scan_timeout_adjustment_multiplier
```

Comment:
Is this already limited to VmOrTemplate here?  I know we do ConatinerImage scanning but maybe that isn't managed by this code, @roliveri ?

We could just say `timeout_adjustment_multiplier = 1` here then let the target override it if it wants.